### PR TITLE
Integrate emojibase

### PR DESCRIPTION
**Pull request by langleyd**
### Context
At the moment web, android and iOS all use different datasources for their emoji picker. This means inconsistencies in what emojis are available(the iOS one is a bit out of date so is missing the new emojis), order they are displayed, different tags etc.

It can also result in bugs like this where the data sent by each client may vary(incorrect unicode points):
image

https://github.com/milesj/emojibase that web uses seems to make most sense, it provides just the json and is spec compliant(generated directly from http://unicode.org/Public/emoji/). It also has translations which we could take advantage of for tags, so that the picker search works for people in non-english languages.

https://github.com/matrix-org/emojibase-bindings is a new repo that will provide bindings for the 3 platforms.

We now have moose!

### CODE REVIEW

File: ElementX/Sources/Services/Emojis/EmojiProvider.swift [hunk 1]
```diff
@@ -74,3 +75,26 @@ class EmojiProvider: EmojiProviderProtocol {
...
+
+extension EmojibaseDatasource: EmojiLoaderProtocol {
+    func load() async -> [EmojiCategory] {
+        do {
+            let store: EmojibaseStore = try await load()
+            return EmojibaseCategory.allCases.map { category in
+                let emojis = store.categories[category.rawValue] ?? []
+                return EmojiCategory(id: category.rawValue, emojis: emojis.compactMap(EmojiItem.init(from:)))
+            }
+        } catch { }
```
Comment (stefanceriu) [inline ElementX/Sources/Services/Emojis/EmojiProvider.swift:87]: We should probably do something with these errors, at least log them if nothing else

**Review by stefanceriu — APPROVED** [review]
Looks good to me, very nice 🤩

**Comment by stefanceriu** [comment]
> searching `wizard` on web doesn't return 🧙, instead you need to search for `mage` which is totally undiscoverable).

I think that's more of a problem with the current implementation than with the data itself. If you look at the generated json you see this

```

group: 1,
hexcode: "1F9D9",
label: "mage",
order: 1723,
tags: [
"sorcerer",
"sorceress",
"witch",
"wizard"
],
unicode: "🧙",
```

**Comment by pixlwave** [comment]
Ok nice, then bring on the Moose!

**Review by pixlwave — APPROVED** [review]
🚢 it!

edit: Although the EmojiProviderTests are failing.

**Pull request merged**